pkg/wireguard/agent/agent.go

```diff
@@ -93,6 +103,37 @@ func NewAgent(privKeyPath string, localNodeStore *node.LocalNodeStore) (*Agent,
 		return nil, err
 	}
 
+	isMaster := false
```

Comment:
this block of code 's target was try to obtain the topology of cluster, maybe this code could be added in other place but needn't tie the wireguard backend so tightly and this block of code depend with inClusterConfiguration, I guess if there is more better ways